packages/panels/src/Commands/MakePageCommand.php

```diff
@@ -70,10 +70,20 @@ public function handle(): int
             )] : Arr::first($panels);
         }
 
+        $resourceDirectories = $panel->getResourceDirectories();
+        $resourceNamespaces = $panel->getResourceNamespaces();
+
+        foreach ($resourceDirectories as $resourceIndex => $resourceDirectory) {
+            if (str($resourceDirectory)->startsWith(base_path('vendor'))) {
+                unset($resourceDirectories[$resourceIndex]);
+                unset($resourceNamespaces[$resourceIndex]);
+            }
+        }
+
         $resourceInput = $this->option('resource') ?? suggest(
             label: 'Which resource would you like to create this in?',
             options: collect($panel->getResources())
-                ->filter(fn (string $namespace): bool => str($namespace)->contains('\\Resources\\'))
+                ->filter(fn (string $namespace): bool => str($namespace)->contains('\\Resources\\') && str($namespace)->startsWith($resourceNamespaces))
```

Comment:
Can you explain this please?

Reply:
In a plugin where it register resources `->resources(...)`, when using the `filament:page` command it will list all resources including in vendor. this added check only allow non-vendor resource namespaces

before 
<img width="468" alt="image" src="https://github.com/user-attachments/assets/2ade8964-ca5b-49e4-86eb-7882099a4cbb">

after
<img width="478" alt="image" src="https://github.com/user-attachments/assets/5aa397b8-502c-44be-9478-faa3a3a38939">

Reply:
Does this also need to happen for resource widgets?

Reply:
not really, the whole codebase only this command has this `suggest`

Reply:
Alright thanks